testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -132,6 +132,8 @@ class SettingsMenuActivity : BaseActivity() {
             QASettingsActivity.start(this@SettingsMenuActivity)
         }
 
+        binding.clearProjectId.setOnClickListener { binding.editProjectId.setText("") }
+        binding.clearSubscriptionKey.setOnClickListener { binding.editSubscriptionKey.setText("") }
```

Comment:
If you want to stick with this solution I suggest this change:

```suggestion
        binding.clearProjectId.setOnClickListener { binding.editProjectId.getText().clear() }
        binding.clearSubscriptionKey.setOnClickListener {  binding.editProjectId.getText().clear() }
```

Reply:
Updated.